client/src/components/Masthead/Masthead.test.js

```diff
@@ -136,4 +143,24 @@ describe("Masthead.vue", () => {
     it("should load webhooks on creation", async () => {
         expect(wrapper.find("#extension a").text()).toBe("Extension Point");
     });
+
+    it("shows link to interactive tools if any is active", async () => {
+        setActivePinia(testPinia);
+        const entryPointStore = useEntryPointStore();
+        entryPointStore.entryPoints = [
+            {
+                model_class: "InteractiveToolEntryPoint",
+                id: "52e496b945151ee8",
+                job_id: "52e496b945151ee8",
+                name: "Jupyter Interactive Tool",
+                active: true,
+                created_time: "2020-02-24T15:59:18.122103",
+                modified_time: "2020-02-24T15:59:20.428594",
+                target: "http://52e496b945151ee8-be8a5bee5d5849a5b4e035b51305256e.interactivetoolentrypoint.interactivetool.localhost:8080/ipython/lab",
+            },
+        ];
+        // avoid race condition with store's reactivity
+        await new Promise((_) => setTimeout(_, 10));
```

Comment:
Can you tell me more about the race condition here?

Reply:
I have nothing elaborate here. If the timeout is not there the wrapper on the next line will not be visible yet. There does not seem to be anything explicitly `async` that we can attach to either.